account_payment_show_invoice/models/account_payment.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountPayment(models.Model):
+    _inherit = 'account.payment'
+
+    @api.multi
+    def _compute_invoice_vendor_references(self):
```

Comment:
Here, I think the reason to use multi is oppose of using ensure_one() here isn't it? We use multi because we want to compute once for all records.
Please correct me if I am wrong.

Reply:
When I double think, should that not be @api.depends('invoice_ids')?
Because the field should be changed in case the 'invoice_ids' are changed.

Reply:
ok, you are right, after we discussed it I agree that @api.multi without ensure_one is fine. : ))